pipelines/build/openjdk10_pipeline.groovy

```diff
@@ -1,3 +1,6 @@
+import groovy.json.JsonSlurper
```

Comment:
Probably for a later PR but I see a lot of repeated code in this openjdk<version>_pipeline.groovy files.

Reply:
Existing issue to do just that -> https://github.com/AdoptOpenJDK/openjdk-build/issues/2290